addon/components/frost-textarea.js

```diff
@@ -96,7 +96,7 @@ export default Component.extend(FrostEventsProxyMixin, {
       yield timeout(200) // Duration of the visibility animation
     }
     this.set('isClearEnabled', showClear)
-  }).restartable(),
+  }).enqueue(),
```

Comment:
It would be best to correct the `_clear` task so that it does cause the `_showClear` task to be initiated twice.  This PR is fixing the symptom, not the root cause.